2025-04-04 15:42:08.866 [info] 




2025-04-04 15:42:08.948 [info] Extension host agent started.
2025-04-04 15:42:09.332 [info] [<unknown>][94e315a5][ManagementConnection] New connection established.
2025-04-04 15:42:09.340 [info] [<unknown>][0d22f776][ExtensionHostConnection] New connection established.
2025-04-04 15:42:09.691 [info] [<unknown>][0d22f776][ExtensionHostConnection] <255841> Launched Extension Host Process.
2025-04-04 15:42:17.617 [error] CodeExpectedError: Could not find pty 4 on pty host
    at I.W (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:12275)
    at I.updateTitle (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:7722)
    at O.s.<computed> (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2962)
    at Object.call (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at el.s (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80918)
    at el.q (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80441)
    at ds.value (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:79843)
    at x.B (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at x.fire (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.C (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:24:29877)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:950:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-04-04 15:42:17.636 [error] CodeExpectedError: Could not find pty 4 on pty host
    at I.W (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:12275)
    at I.updateIcon (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:7772)
    at O.s.<computed> (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2962)
    at Object.call (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at el.s (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80918)
    at el.q (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80441)
    at ds.value (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:79843)
    at x.B (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at x.fire (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.C (file:///home/aituter/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:24:29877)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:950:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-04-04 15:47:08.957 [info] New EH opened, aborting shutdown
2025-04-06 01:25:28.232 [warning] No ptyHost heartbeat after 6 seconds
2025-04-06 01:26:25.277 [warning] No ptyHost heartbeat after 6 seconds
2025-04-06 01:27:05.907 [warning] No ptyHost heartbeat after 6 seconds
2025-04-06 01:27:17.288 [warning] No ptyHost heartbeat after 6 seconds
2025-04-06 01:27:21.217 [info] [<unknown>][0d22f776][ExtensionHostConnection] The client has reconnected.
2025-04-06 01:27:38.827 [warning] No ptyHost heartbeat after 6 seconds
2025-04-06 01:27:47.856 [info] [<unknown>][0d22f776][ExtensionHostConnection] <255841> Extension Host Process exited with code: null, signal: SIGKILL.
2025-04-06 01:27:47.867 [info] Last EH closed, waiting before shutting down
2025-04-06 01:27:47.984 [error] [<unknown>][0d22f776][ExtensionHostConnection] Unknown reconnection token (seen before).
2025-04-06 01:27:48.491 [info] [<unknown>][22457abd][ExtensionHostConnection] New connection established.
2025-04-06 01:27:48.517 [info] [<unknown>][22457abd][ExtensionHostConnection] <464984> Launched Extension Host Process.
